pyproject.toml

```diff
@@ -11,7 +11,7 @@ packages = [
 keywords = ["openapi", "pydantic", "code-generator", "openapi-codegen"]
 license = "MIT"
 classifiers = [
-    "Development Status :: 5 - Production/Stable",
+    "Development Status :: 3 - Alpha",
```

Comment:
I'd keep it at 5 personally, but up to you :)

Reply:
We're changing a lot, especially in the public API. Lets keep it as alpha for now.